pkg/kubelet/kubelet_node_status.go

```diff
@@ -369,6 +369,7 @@ func (kl *Kubelet) syncNodeStatus() {
 		// This will exit immediately if it doesn't need to do anything.
 		kl.registerWithAPIServer()
 	}
+	kl.updateRuntimeUp()
```

Comment:
In Kubelet#fastStatusUpdateOnce, 
```
			kl.updateRuntimeUp()
			kl.syncNodeStatus()
```
I am not sure adding extra call to kl.updateRuntimeUp() here is necessary.

Reply:
Sorry  about I reply late and I have push a new commit on this issue.

In kl.updateRuntimeUp(), it will check runtime state and update runtime status.  
The init stauts of  runtimeState is:
`
func newRuntimeState(
	runtimeSyncThreshold time.Duration,
) *runtimeState {
	return &runtimeState{
		lastBaseRuntimeSync:      time.Time{},
		baseRuntimeSyncThreshold: runtimeSyncThreshold,
		networkError:             ErrNetworkUnknown,
	}
}
`
We must notify that networkError is not nil  and  lastBaseRuntimeSync is time.Time{}.
If we restart kubelt, kubelet will exec kl.registerWithAPIServer() and kl.initialNode(context.TODO()).
In  kl.initialNode will check runtime status and find it have error(networkerror and lastBaseRuntimeSync behind time.Now greater than baseRuntimeSyncThreshold).So this node status will be NotReady and post to APIServer. But This node is real ready.

 kl.updateRuntimeUp() will exec in a Goroutine and is sometimes can not exec before kl.initialNode.

And I add kl.updateRuntimeUp() before kl.registerWithAPIServer() to ensure runtime real state.